tokio/src/sync/semaphore.rs

```diff
@@ -680,3 +682,10 @@ impl Drop for OwnedSemaphorePermit {
         self.sem.add_permits(self.permits as usize);
     }
 }
+
+impl Semaphore {
+    /// The maximum number of permits which a semaphore can hold.
+    ///
+    /// Exceeding this limit typically results in a panic.
+    pub const MAX_PERMITS : usize = super::batch_semaphore::Semaphore::MAX_PERMITS;
+}
```

Comment:
Perhaps this should go at the top of the existing impl block to better display it on the docs?

Reply:
Moved and fixed fmt.